ecosystem/sep-0006.md

```diff
@@ -902,7 +903,7 @@ Request parameters:
 Name | Type | Description
 -----|------|------------
 `asset_code` | string | The code of the asset of interest. E.g. BTC, ETH, USD, INR, etc.
-`account` | string | The stellar account ID involved in the transactions.
+`account` | string | The stellar account ID involved in the transactions. If the service requires SEP-10 authentication, this parameter match the authenticated account.
```

Comment:
There's a word missing in this sentence, "this parameter match the authenticated account." Should it be "may not match" or "will match"?

Reply:
Ah thanks for pointing that out. It should say "must match". Clients should only be able to see records of transactions they initiated.

Reply:
Makes sense. 👍🏻